src/blockdata/transaction.rs

```diff
@@ -672,7 +672,7 @@ impl fmt::Display for SigHashType {
     }
 }
 
-impl str::FromStr for SigHashType {
+impl core::str::FromStr for SigHashType {
```

Comment:
How this was working with no_std before?

Reply:
No clue, I was surprised as well.